lib/timezonedata/windowszones.php

```diff
@@ -10,7 +10,7 @@
  * @license http://sabre.io/license/ Modified BSD License
  */
 
-return  [
+return [
```

Comment:
This file says that it is auto-generated. But it has not been updated from the "source" any time recently.
php-cs-fixer did not like having 2 spaces here, so I made the edit anyway.
If someone re-generates the file from upstream some day, they might need to do a small edit like this to keep php-cs-fixer happy.

Reply:
Oh, actually it is "auto-generated" from out own script. https://github.com/sabre-io/vobject/blob/master/bin/fetch_windows_zones.php

So it does not matter. I will update that script also...